app/main.py

```diff
@@ -75,6 +79,17 @@ def post():
     return app
 
 
+def zip(buffer):
```

Comment:
Can you please name it `is_zip_file` or something? This feels extra confusing because of https://docs.python.org/3.3/library/functions.html#zip